Documentation/ceph-object-store-crd.md

```diff
@@ -141,6 +141,7 @@ All the other settings from the gateway section will be ignored, except for `sec
 The [zone](ceph-object-multisite.md) settings allow the object store to join custom created [ceph-object-zone](ceph-object-multisite-crd.md).
 
 * `name`: the name of the ceph-object-zone the object store will be in.
+* `endpoint`: the endpoint of the object-store to be added to the zone's endpoint list.
```

Comment:
When should users configure this?

Reply:
endpoint should be `endpoints` plural and accept a list, as that's what is accepted in radosgw: https://docs.ceph.com/en/latest/radosgw/multisite/#create-a-master-zone

Reply:
The reason a user would want to set this option is if they have a ceph cluster deployed in 2 different k8s clusters (or 1 rook cluster and one non-rook cluster elsewhere).  Service IPs aren't typically routable outside the cluster, so without these endpoints no replication will actually take place.

Reply:
@mgugino-uipath The reason `endpoint` is singular not plural is that the value should correspond to whatever the endpoint is of that object-store only. By adding this value in for the object-store, the zone and zonegroup's endpoint lists get appended (or removed) automatically. 

To be clear, what's stored internally at the zone and zonegroup level is still a list of all the endpoints, that list is just being added in one at a time.